README.md

```diff
@@ -15,55 +15,68 @@
-| [Deploy Transformers models with confidentiality](https://blog.mithrilsecurity.io/transformers-with-confidentiality/)                | Sentiment analysis                      | DistilBERT      | Transformers |
-| [Confidential medical image analysis with COVID-Net and BlindAI](https://blog.mithrilsecurity.io/confidential-covidnet-with-blindai/) | Chest XRAY analysis for COVID detection | COVID-Net-CXR-2 | Deep CNN     |
+### Step 1 - Deploying the server
 
-## Getting started
+Since the BlindAI server relies on specific hardware (Intel SGX) for security, this Getting Started guide will focus on running the Simulation mode, which can run on any machine. Note that the Simulation mode is not secure.
+
+To learn about deploying BlindAI on real hardware, see the [Deploy on Hardware](TODO) documentation page and skip the next step. Here is also a [step-by-step guide to create an SGX-enabled Azure VM and deploy BlindAI in 5 minutes](TODO).
 
-To deploy a model on sensitive data, with end-to-end protection, we provide a *Docker* image to serve models with confidentiality, and a *client SDK* to consume this service securely.
+Run the Simulation docker image.
 
-### Note
+```bash
+docker run -it -p 50051:50051 -p 50052:50052 mithrilsecuritysas/blindai-server-sim
+```
 
-Because the server requires specific hardware, for instance *Intel SGX* currently, we also provide a *simulation mode*. Using the *simulation mode*, any computer can serve models with our solution. However, the two key properties of secure enclaves, data in use confidentiality, and code attestation, will not be available. **Therefore this is just for testing on your local machine but is not relevant for real guarantees in production**.
+### Step 2 - Uploading the model to the server
 
-Our first article [Deploy Transformers with confidentiality](https://blog.mithrilsecurity.io/transformers-with-confidentiality) covers the deployment of both simulation and hardware mode. 
+The Python client SDK has a very simple API, but it deals with most of the complexity of working with Confidential Computing. When connecting to the BlindAI server, the client will ask for a _quote_. This cryptographic _quote_ enables the client to know whether it is really talking with an Intel SGX enclave and that the loaded binary is indeed a known one. The hardware then guarantees privacy by completely isolating the server code and memory from the rest of the system.
 
-### A - Deploying the server
+You can learn more about the attestation mechanism for code integrity [here](https://sgx101.gitbook.io/sgx101/sgx-bootstrap/attestation).
 
-Deploy the inference server, for instance using one of our *Docker* images. To get started quickly, you can use the image with simulation, which does not require any specific hardware. 
+#### Install the python library
+
+If you feel extra-lazy, we also have a Jupyter notebook environment you can just pull and use, just run
```

Comment:
i am not too keen on this wording
this may need to move in another section, but I couldn't find any other place that feels natural